### PR TITLE
Set up Cypress

### LINKED ISSUE #46: Set up Cypress
Set up Cypress for end to end tests to help meet the expected level of test coverage.
Follow up issue to start structuring the tests: https://github.com/button-inc/digital_marketplace/issues/47

### PR DESCRIPTION
**Pull request by BCerki**
This PR:
- installs Cypress, including example tests
- installs eslint plugins for Cypress

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (mikevespi) [inline cypress.json]: Can we add:
- baseurl (can be localhost:3000)
- video set to false

**Comment by BCerki** [comment]
I had to additionally set some mocha rules to "warn" to allow the cypress tests to pass. This suggests that the mocha eslint is scanning the cypress tests--is that going to be a problem? Presumably some projects use both mocha and cypress.

**Comment by mikevespi** [comment]
I think that it might have to do with the advanced examples that are included. Would it be possible to write a "placeholder" test so to speak that just hits our app and maybe does one thing just to give make sure it's working there, and once thats running get rid of the examples? I have a feeling that will clear that up and also allow us to avoid having a bunch of example tests in the repo that run in CI.

**Review by BryanGK — DISMISSED** [review]
That's pretty cool to see the test runner do its thing. Can I request that some docs are included to know how to run cypress? Also would it be a good idea to have the `npm run cypress:open` cmd also include building and running the app? Maybe @wenzowski can weigh in on that

**Comment by BryanGK** [comment]
I pulled down the branch and got this error:
Screen Shot 2021-12-10 at 1 39 29 PM
Have you brought in changes from `main` that get rid of those files?

**Review by BryanGK — APPROVED** [review]
Test fired up the app and ran the test. Looks good to me.

**Pull request merged**